src/main/scala/org/clulab/coref/CorefUtils.scala

```diff
@@ -62,7 +62,7 @@ object CorefUtils {
   def argsComplete(args: Map[String,Seq[CorefMention]], lbls: Seq[String]): Boolean = {
     lbls match {
       case binding if lbls contains "Binding" =>
-        args.contains("theme") && args("theme").length == 2
+        args.contains("theme") && args("theme").length >= 2 // Binding is not required to be binary anymore (see binding_token_5)
```

Comment:
@danebell I had to change this check to support bindings with more than two participants